devnet/p2w-attest.yaml

```diff
@@ -45,11 +45,35 @@ spec:
             - name: P2W_EXIT_ON_ERROR
               value: "true"
           tty: true
-          readinessProbe:
-            tcpSocket:
-              port: 2000
-            periodSeconds: 1
-            failureThreshold: 300
+          # Probes, in order of appearance https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+          #
+          # Startup probe - delays other probes until it gets its first success
+          startupProbe:
+            httpGet:
+              path: /healthcheck
+              port: 3000
+            failureThreshold: 100 # up to 100 * 10 seconds to report initial healthy status
```

Comment:
i don't understand this comment. why the 10 seconds part?

Reply:
it will try 100 times, waiting 10 seconds (periodSeconds) in between, for a total of 100 * 10 seconds to become ready and move on to the next probe.

Reply:
I think here `initialDelaySeconds` is cleaner.

Reply:
The initial delay will not poll the service, incurring a fixed delay instead. This check is not expensive, I would not do that.

Reply:
Makes sense. 👍